### DISCUSSION TITLE
finalise a list of supported reconstruction models

### DISCUSSION THREAD

**Discussion by michaelchin** [GPlates/gplately#217] [General]
We need to make an official list of supported reconstruction models and update the doc accordingly.
I will bring this matter up in the next auscope meeting.

Let's use the table below to finalise the list.  Order by year first and then alphabet. 

| **Model name string Identifier** | **Zenodo** |**Topology features** | **Static polygons** | **Coast-lines** | **Cont-inents** | **COB** | **Age grids** | **SR grids** |
|:--------------------------------:|:--------------:|:---------------------:|:-------------------:|:---------------:|:---------------:|:--------:|:-------------:|:------------:|
|  Alfonso2024                  |       ✅       |          ✅           |          ✅          |        ✅        |        ❌        |     ❌    |       ❌       |       ❌      |
|  Cao2024                        |       ✅       |          ✅           |          ✅          |        ✅        |         ✅       |     ✅    |       ❌       |       ❌      |
|  Muller2022                    |        ✅      |           ✅           |          ✅          |        ✅        |        ✅        |     ✅    |       ❌       |       ❌      |
| Zahirovic2022                |        ✅      |           ✅           |          ✅          |        ✅        |        ✅        |     ❌    |       ✅       |       ✅      |
|  Merdith2021                  |        ✅      |           ✅           |          ✅          |        ✅        |        ✅        |     ❌    |       ❌       |       ❌      |
|  Clennett2020                |        ✅      |           ✅           |          ✅          |        ✅        |        ❌       |     ❌    |       ✅       |       ✅      |
|  Clennett2020_M2019  |        ✅      |           ✅           |          ✅          |        ✅        |        ❌       |     ❌    |       ✅       |       ✅      |
|  Clennett2020_S2013   |        ✅      |           ✅           |          ✅          |        ✅        |        ❌        |     ❌    |       ❌       |       ❌      |
|  Muller2019                    |        ✅      |           ✅           |          ✅          |        ✅        |        ✅        |     ✅    |       ✅       |       ❌      |
|  Young2018                    |        ✅      |           ✅           |          ✅          |        ✅        |        ✅        |     ❌    |       ❌       |       ❌      |
|  TorsvikCocks2017        |        ❌      |           ❌           |          ✅           |        ✅        |        ❌        |     ❌    |       ❌       |       ❌      |
|  Matthews2016              |        ✅      |           ✅           |          ✅          |        ✅        |        ✅        |     ❌    |       ❌       |       ❌      |
|  Matthews2016_pmag_ref |  ❌      |            ❌           |          ✅          |        ✅        |        ✅        |     ❌    |       ❌       |       ❌      |
|  Muller2016                    |        ✅       |           ✅           |          ✅          |        ✅        |        ❌        |     ✅    |       ✅       |       ❌      |
|  Scotese2016                 |        ✅     |             ❌           |          ✅          |        ✅        |        ❌        |     ❌    |       ❌       |       ❌      |
|  Zahirovic2016               |        ✅       |           ✅           |          ✅          |        ✅        |        ✅        |     ❌    |       ❌       |       ❌      |
|  Gibbons2015               |          ✅       |           ✅           |          ✅          |        ✅        |        ❌        |     ❌    |       ❌       |       ❌      |
|  Zahirovic2014              |        ✅       |           ❌           |          ✅          |        ✅        |        ❌        |     ❌    |       ❌       |       ❌      |
|  Shephard2013              |        ✅       |           ✅           |          ✅          |        ✅        |        ❌        |     ❌    |       ❌       |       ❌      |
|  Gurnis2012                    |        ✅        |          ✅           |          ✅            |       ✅        |        ❌        |     ❌   |       ❌       |       ❌      |
|  Seton2012                    |         ✅       |           ✅           |          ✅           |        ✅        |         ✅       |     ✅    |       ✅       |       ❌      |
|  Muller2008                   |        ❌       |             ❌           |          ✅          |        ❌        |        ❌        |     ❌    |       ❌       |       ❌      |

Change log:
- add Gibbons2015 and Zahirovic2014 (2024-10-24 by MC)
- add back Muller2008 (2024-10-24 by MC)
- add Zahirovic2022  (2024-10-17 by MC)
- add Cao2024  (2024-10-16 by MC)
- remove Pehrsson2015 approved by DM in email "Pehrsson2015 model" (2024-08-09 by MC)
- remove Cao2020 approved by DM in email "cao2020 model" (2024-08-09 by MC)
- remove Muller2008 approved by DM in email "Muller2008" (2024-08-08 by MC)
- add Gurnis2012 (2024-08-06 by MC)
- add Zahirovic2016 (2024-08-02 by MC)
- add matthews2016_pmag_ref model;; see email 'MATTHEWS2016' (2024-07-24 by MC)
-  harvest Matthews2016 files from Zenodo (2024-07-18 by MC)
- Young2019 should be Young2018.  harvest Young2018 files from Zenodo (2024-07-18 by MC)
- Drop Merdith2017 (2024-07-18 approved by DM in email "Merdith2017")
- harvest Clennett2020,  Clennett2020_M2019 and Clennett2020_S2013 files from Zenodo (2024-07-18 by MC)
- harvest Merdith2021 files from Zenodo (2024-07-18 by MC)
- remove **Rot. files** column, all models must have rotation file  (2024-07-18 by MC)
- harvest Muller2022 files from Zenodo (2024-07-18 by MC)
- order the list by year first and then alphabet (2024-07-18 by MC)
- add Alfonso2024
- drop Scotese2008 (in email Scotese2016 and Scotese2008)
- drop Li2008 (in email Merdith2017 and Li2008)
- Golonka (undecided yet)

**Comment by brmather** [comment]
Talk to Bianca (RA) about this as she is migrating all the plate models to Zenodo.

**Comment by michaelchin** [comment]
This issue will also close https://github.com/GPlates/gplately/issues/149

**Comment by michaelchin** [comment]
add new model "Alfonso2024" -- https://zenodo.org/records/11392269

**Comment by michaelchin** [comment]
I have sent email "Golonka" to customers regarding the Golonka models in GPlately and plate model manager. One of the customers confirmed that the original Golonka2007 is different from the Golonka model in plate model manager. 

Waiting for more specific instructions from customers regarding "Golonka" model. Do nothing until having got further instructions from customers.

**Comment by michaelchin** [comment]
I have sent an email "Reconstruction models on Zenodo" to the person who is in charge of uploading models to Zenodo for a list of models on Zenodo. Waiting for the list.

**Comment by michaelchin** [comment]
- [Muller_etal_2022_SE](https://url.au.m.mimecastprotect.com/s/VgDKCNLJyQUVRrjw5CmylNZ?domain=zenodo.org): A tectonic-rules-based mantle reference frame since 1 billion years ago – implications for supercontinent cycles and plate–mantle system evolution (latest version 1.2.2)

 

- [Merdith_etal_2021_ESR](https://url.au.m.mimecastprotect.com/s/UVtUCOMKzVT5G9vEruvzony?domain=zenodo.org): Extending full-plate tectonic models into deep time: Linking theNeoproterozoic and the Phanerozoic (latest version 1.2.2)

 

- [Clennett_etal_2020_G3](https://url.au.m.mimecastprotect.com/s/N_2VCP7LAXfvlr3JAC17rYr?domain=zenodo.org): A Quantitative Tomotectonic Plate Reconstruction of Western North America and the Eastern Pacific Basin

 

- [Muller_etal_2019_Tectonics](https://url.au.m.mimecastprotect.com/s/anBGCQnMBZfBxEoJ4Hk7VIF?domain=zenodo.org):  A Global Plate Model Including Lithospheric Deformation Along Major Rifts and Orogens Since the Triassic (latest version 3.0)

 

- [Young_etal_2018_GeosciFrontiers](https://url.au.m.mimecastprotect.com/s/QMT3CROND2unVk5yPHqMQ7P?domain=zenodo.org): Global kinematics of tectonic plates and subduction zones since the late Paleozoic Era

 

- [Merdith_etal_2017_GR](https://url.au.m.mimecastprotect.com/s/BDfcCVARKgC0vLgzjckyLXe?domain=zenodo.org): A full-plate global reconstruction of the Neoproterozoic

 

- [Matthews_etal_2016_GPC](https://url.au.m.mimecastprotect.com/s/ST2RCWLVXkUzkgDNwS1HtC6?domain=zenodo.org): Global plate boundary evolution and kinematics since the late Paleozoic

 

- [Zahirovic_etal_2016_ESR](https://url.au.m.mimecastprotect.com/s/S85RCXLW2mUBJyOk7HMO4jv?domain=zenodo.org): Tectonic evolution and deep mantle structure of the eastern Tethys since the latest Jurassic

 

- [Muller_etal_2016_AREPS](https://url.au.m.mimecastprotect.com/s/HitbCYW8Nock8JAjou2Th1V?domain=zenodo.org): Ocean basin evolution and global-scale plate reorganization events since Pangea breakup (latest version 1.17)

 

- [Zahirovic_etal_2014_SE/Gibbons_etal_2015_GR](https://zenodo.org/records/10595659): A tectonic model reconciling evidence for the collisions between India, Eurasia and intra-oceanic arcs of the central-eastern Tethys (awaiting approval)

 

- [Shephard_etal_2013_ESR](https://url.au.m.mimecastprotect.com/s/CYOmCZY1NqiPGVQXnCJ-Wie?domain=zenodo.org): The tectonic evolution of the Arctic since Pangea breakup: Integrating constraints from surface geology and geophysics with mantle structure

 

- [Seton_etal_2012_EPSL](https://url.au.m.mimecastprotect.com/s/ehuoC1WLPxcnRDqLwHWZhmI?domain=zenodo.org): Global continental and ocean basin reconstructions since 200 Ma

 

- [Shephard_etal_2012_EPSL:](https://url.au.m.mimecastprotect.com/s/9e7OC2xMQziKjJ80ou85sNa?domain=zenodo.org) Testing absolute plate reference frames and the implications for the generation of geodynamic mantle heterogeneity structure

 

- [Gurnis_etal_2012_CompGeosci](https://url.au.m.mimecastprotect.com/s/LIUZC3QNPBi7BJ2WYcznmp_?domain=zenodo.org): Plate Tectonic Reconstructions with Continuously Closing Plates

 

- [Scotese_PaleoAtlas_2016_v3](https://url.au.m.mimecastprotect.com/s/NBg8C4QOPEiYoQlRAcLAK9F?domain=zenodo.org): PALEOMAP PaleoAtlas for GPlates (tutorial)

 

- [Alfonso_etal_2024_modClennettMuller](https://url.au.m.mimecastprotect.com/s/pFsTC5QPXJigvQ6wYuQvkd2?domain=zenodo.org): A Modified Global Plate Model Including Quantitative Tomotectonic Plate Reconstruction of Western North America and the Eastern Pacific Basin (standalone model for an already existing Zenodo upload)

**Reply by michaelchin** [reply]
> * [Zahirovic_etal_2014_SE/Gibbons_etal_2015_GR](https://zenodo.org/records/10595659): A tectonic model reconciling evidence for the collisions between India, Eurasia and intra-oceanic arcs of the central-eastern Tethys (awaiting approval)

Does anyone have an update on the "awaiting approval" status?

@GPlates/gplately-dev

**Comment by michaelchin** [comment]
This issue should be a discussion. Convert to discussion.

**Comment by michaelchin** [comment]
It seems that CEED6.zip goes by name "TorsvikCocks2017" in gplately. Not sure what to do about it.

**Comment by nickywright** [comment]
Can we please add the Zahirovic et al. 2022 model to this list? 
It's the model that has come with the GPlates geodata for a couple releases and seems weird to not be here already...

**Reply by michaelchin** [reply]
> Can we please add the Zahirovic et al. 2022 model to this list? It's the model that has come with the GPlates geodata for a couple releases and seems weird to not be here already...

Please bring this up to DM. It is DM's responsibility to decide which models to use.

**Comment by nickywright** [comment]
And why is Müller et al. 2008 getting removed? Can we please add that back (this is a key model for paleogeography reconstructions so should be included IMO...)

**Reply by michaelchin** [reply]
> And why is Müller et al. 2008 getting removed? Can we please add that back (this is a key model for paleogeography reconstructions so should be included IMO...)

The text below is what DM told me. And he agreed to remove Muller2008. Could you please double check with DM? Thanks.
 
_The oldest proper plate model we have that was published was Seton et al. (2012).
There is no coherent plate model from 2008. Roughly, that model corresponds to what was later published in Gurnis et al. (2012), but I would not bother with that one._

**Comment by michaelchin** [comment]
**TODO**:

- bring back Müller et al. 2008 
- add Zahirovic et al. 2022

**Extremely important!!! DO NOT FORGET!!!**

**Comment by michaelchin** [comment]
a bit update about Zahirovic et al. 2022. 

The model had been added to the new "plate model manager" in last December according to version info in models.json. So, it means we have already can use it in GPlates since last December. And it had even been set as the "default" model, which means you can "PlateModelManager().get_model("default")" to get the  Zahirovic et al. 2022. .

<img width="894" alt="Screenshot 2024-10-06 at 2 29 53 PM" src="https://github.com/user-attachments/assets/e8d27258-8ccc-4f55-952f-77a955e67d2e">

The problem is that we have not updated the online documentation yet. And when I started this discussion to finalise the model list for GPlately 2.0, Zahirovic et al. 2022 was not in the list of models on Zenodo. We agreed that all models should be put on Zenodo and I was told to add and document models according to the list of models on Zenodo.

**Reply by michaelchin** [reply]
> a bit update about Zahirovic et al. 2022.
> 
> The model had been added to the new "plate model manager" in last December according to version info in models.json. So, it means we have already can use it in GPlates since last December. And it had even been set as the "default" model, which means you can "PlateModelManager().get_model("default")" to get the Zahirovic et al. 2022. .
> 
> <img alt="Screenshot 2024-10-06 at 2 29 53 PM" width="894" src="https://private-user-images.githubusercontent.com/2688316/373919728-e8d27258-8ccc-4f55-952f-77a955e67d2e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjgxODYxMDMsIm5iZiI6MTcyODE4NTgwMywicGF0aCI6Ii8yNjg4MzE2LzM3MzkxOTcyOC1lOGQyNzI1OC04Y2NjLTRmNTUtOTUyZi03N2E5NTVlNjdkMmUucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTAwNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEwMDZUMDMzNjQzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmMwM2JhMGFhMzk4ZTc5ZGRjMDM2Y2QwYjA1YmMzNmEwYjhmYmY2MWQ1Y2ZmZTcyYjQ0YTM1MzE3NTVjYWNiNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.d7G06xJA_Qs6RxGbvRY-P0IdKaY6u7veDc-M0kKHk7Y">
> The problem is that we have not updated the online documentation yet. And when I started this discussion to finalise the model list for GPlates 2.0, Zahirovic et al. 2022 was not in the list of models on Zenodo. We agreed that all models should be put on Zenodo and I was told to add and document models according to the list of models on Zenodo.

@nickywright NW might be interested in this update.

**Comment by michaelchin** [comment]
Remember to ask the RA about  the "awaiting approval" thing on [Zahirovic_etal_2014_SE/Gibbons_etal_2015_GR](https://zenodo.org/records/10595659):

And also ask the RA if Zahirovic et al. 2022 is on Zenodo.

**Comment by michaelchin** [comment]
Need to be extremely careful with these models. If I made any mistake, no matter if it is my fault or how small the mistake is, I might get abused by someone. Michael, you need to EXTREMELY CAREFUL!!!!

**Reply by michaelchin** [reply]
> EXTREMELY

Micheal, I think you are imaging things. I don't think the hostile work environment exists. Clear your head and think straight.  -- XQ

**Comment by michaelchin** [comment]
Zahirovic_etal_2022 model:

https://zenodo.org/records/4729046

Cao2024 model:

https://zenodo.org/records/13628813

And update https://www.earthbyte.org/category/resources/data-models/global-regional-plate-motion-models/ (waiting for decision)

**Comment by michaelchin** [comment]
Dig up an old email. Need to think about how to deal with these models.

-------------------------------------------------

I found an undocumented model "Golonka2007" in GPlately's DataServer. The data is from the supplementary of [this paper](https://www.cambridge.org/core/journals/geological-magazine/article/abs/palaeolatitudinal-distribution-of-lithologic-indicators-of-climate-in-a-palaeogeographic-framework/3DB4F8A0E6B424CC9DB3B6166B4813DD#article). See the link and screenshot below. I don't know why it is undocumented. Shall I keep it and document it properly?

And also, there is a "Golonka" model in the plate-model-manager. After some digging, I found that this "Golonka" model is in fact from [Nicky's paper](https://bg.copernicus.org/articles/10/1529/2013/bg-10-1529-2013.html). I don't know how different this one is from "Golonka2007". Should I keep this one and rename it to "Wright2013"?


"Golonka2007":  https://static.cambridge.org/content/id/urn:cambridge.org:id:article:S0016756818000110/resource/name/S0016756818000110sup001.zip



"Golonka" in the plate-model-manager:
https://www.earthbyte.org/webdav/ftp/incoming/mchin/plate-models/GOLONKA/

**Comment by michaelchin** [comment]
I should design a GitHub template to allow people to request adding their models into GPlately so that we can keep records. The records could be useful when we need to check the history.
